app/ui/omni-box.js

```diff
@@ -107,6 +109,34 @@ class OmniBox extends HTMLElement {
     this.forwardButton.addEventListener('click', () => {
       this.dispatchEvent(new CustomEvent('forward'))
     })
+
+    // middle mouse click paste&go
+    const win = new BrowserWindow()
+    this.input.addEventListener('paste-and-go', () => {
+      win.webContents.executeJavaScript('window.location.href = ""')
```

Comment:
Instead of using the BrowserWindow to execute JavaScript, you should have the component emit a `navigate` event which will trigger a load of the URL using existing methods. The clue for this is that this is what happens in the "submit" listener around  line 80.

```
      this.dispatchEvent(new CustomEvent('navigate', { detail: { url 
```

Reply:
Right I thought of that too but since it didn't work at first I kinda steered away for a while. I'll look do that